docs/users/api-usage.rst

```diff
@@ -65,7 +65,83 @@ of tracks like this:
     Track: Never Gonna Give You Up, listened at 1443521965
     Track: Never Gonna Give You Up, listened at 42042042
 
+Lookup MBIDs
+------------
 
+To interact with various ListenBrainz features, you will often need a MBID of
+the recording of a listen. You can use the :ref:`metadata-api` endpoints to
+lookup MBID and additional metadata for the listen using its track name and
+artist name. For instance,
+
+.. include:: ./api_usage_examples/lookup_metadata.py
+   :code: python
+
+.. code-block:: bash
+
+  ./lookup_metadata.py
+
+  Please input the track name of the listen: One Last Time
```

Comment:
I find this block confusing, but that may only be because of how it is displayed in a PR. I wonder if stating it like this might be less confusing:

Please provide the prompted data to the script to lookup the given track. Currently the release argument is not used, but we plan to support in the near future, so we encourage you to start sending release information if you have it.

Reply:
Done.